lib/nesta/config.rb

```diff
@@ -91,7 +91,7 @@ def self.yaml_exists?
     private_class_method :yaml_exists?
 
     def self.can_use_yaml?
-      ENV.keys.grep(/^NESTA/).empty? && yaml_exists?
+      yaml_exists?
```

Comment:
We no longer need both `yaml_exists?` and `can_use_yaml?`; they're identical. I'll take a look at nailing one of them.